proc_dash/utility.py

```diff
@@ -28,9 +38,9 @@ def construct_summary_str(data: pd.DataFrame) -> str:
 Total number of unique sessions: {data["session"].nunique()}"""
 
 
-def get_required_bagel_columns() -> list:
+def get_required_bagel_columns(schema_file: str) -> list:
     """Returns names of required columns from the bagel schema."""
-    with open(SCHEMAS_PATH / "bagel_schema.json", "r") as file:
+    with open(SCHEMAS_PATH / schema_file, "r") as file:
         schema = json.load(file)
 
     required_columns = []
```

Comment:
in next line `col_category` is unused. NBD, but check your IDE to see if this gets flagged.